5001-6000/LIT5986CommPsalter.xml

```diff
@@ -41,20 +47,69 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </textClass>
             <langUsage>
                 <language ident="en">English</language>
-                <language ident="gez">Gǝʿǝz</language>
+                <language ident="am">Amharic</language>
             </langUsage>
         </profileDesc>
         <revisionDesc>
             <change who="DR" when="2020-05-22">Created entity</change>
+            <change who="DN" when="2021-01-22">Further work</change>
         </revisionDesc>
     </teiHeader>
     <text>
         <body>
             <div type="bibliography">
             <listRelation>
                 <relation name="saws:isCommentOn" active="LIT5986CommPsalter" passive="LIT2701Dawit"/>
-            </listRelation>
+                <relation name="lawd:hasAttestation" active="LIT5986CommPsalter" passive="ESrqg015"/>
+                <relation name="lawd:hasAttestation" active="LIT5986CommPsalter" passive="BLorient535"/>
+    </listRelation>
+                <div type="edition">
+                  <note>The division into text parts follows <ref type="mss" corresp="BLorient353"/>.</note>
+                   <div type="textpart" subtype="incipit" xml:id="LIT5985CommPsalms">
```

Comment:
please add here `xml:lang="am"` (in the upper div="edition" or in each textpart)

Reply:
done